packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2681,26 +2681,30 @@ String _concatStrings({
 ///    [SemanticsSortOrder] manages.
 ///  * [OrdinalSortKey] for a sort key that sorts using an ordinal.
 class SemanticsSortOrder extends Diagnosticable implements Comparable<SemanticsSortOrder> {
+  /// Creates an object that describes the sort order for a [Semantics] widget.
+  ///
   /// Only one of `key` or `keys` may be specified, but at least one must
   /// be specified. Specifying `key` is a shorthand for specifying
-  /// `keys = <SemanticsSortKey>[key]`.
+  /// `keys: <SemanticsSortKey>[key]`.
   ///
-  /// If [discardParentOrder] is set to true, then the
-  /// [SemanticsSortOrder.keys] will replace the list of keys from the parents
-  /// when merged, instead of extending them.
+  /// If [discardParentOrder] is set to true, then the [SemanticsSortOrder.keys]
+  /// will _replace_ the list of keys from the parents when merged, instead of
+  /// extending them.
   SemanticsSortOrder({
     SemanticsSortKey key,
     List<SemanticsSortKey> keys,
     this.discardParentOrder = false,
-  })
-    : assert(key != null || keys != null, 'One of key or keys must be specified.'),
-      assert(key == null || keys == null, 'Only one of key or keys may be specified.'),
-      keys = key == null ? keys : <SemanticsSortKey>[key];
+  }) : assert(key != null || keys != null, 'One of key or keys must be specified.'),
+       assert(key == null || keys == null, 'Only one of key or keys may be specified.'),
+       keys = key == null ? keys : <SemanticsSortKey>[key];
 
   /// Whether or not this order is to replace the keys above it in the
   /// semantics tree, or to be appended to them.
   final bool discardParentOrder;
 
+  /// The keys that should be used to sort this node.
+  ///
+  /// Typically only one key is provided, using the constructor's `key` argument.
```

Comment:
Well, I wouldn't say it's typical. There are two main ways to use it: either specify the entire string of keys in one go, replacing whatever the hierarchy built, or specify one at each level of the widget hierarchy.

I'd say something like "If only one key is needed, using the constructor's `key` argument is less verbose."

Reply:
I guess I don't really understand how this works. Can you suggest some sample code I could include here to explain what "keys" does?